src/scipp/core/data_group.py

```diff
@@ -51,6 +51,8 @@ class DataGroup(MutableMapping):
     such as positional- and label-based indexing and Scipp operations by mapping them
     to the values in the dict. This may happen recursively to support tree-like data
     structures.
+
+    .. versionadded:: NEXT_RELEASE
```

Comment:
I would mark the placeholder with some special character to make sure that we don't accidentally replace something that was meant to be 'NEXT_RELEASE'. I don't know why we would want that but it seems risky to just do a search-and-replace in the entire codebase.

So something like `$NEXT_RELEASE$` or `{{ NEXT_RELEASE }}` would protect us. Though the latter clashes with Jinja and the former with Python's template module.

Reply:
Renamed to `RELEASE_PLACEHOLDER`